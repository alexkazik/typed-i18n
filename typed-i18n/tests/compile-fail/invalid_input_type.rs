use typed_i18n_derive::TypedI18N;

#[derive(Copy, Clone, TypedI18N)]
#[typed_i18n(filename = "tests/compile-fail/demo.lrc")]
#[typed_i18n(builder = "bool", input = "5+5")]
enum Language {
    En,
}

fn main() {}
