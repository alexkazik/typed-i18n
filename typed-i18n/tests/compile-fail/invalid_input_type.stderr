error: invalid input type: 5+5
 --> tests/compile-fail/invalid_input_type.rs:5:32
  |
5 | #[typed_i18n(builder = "bool", input = "5+5")]
  |                                ^^^^^
